resources/europe/italy/it-discourse.json

```
@@ -0,0 +1,15 @@
+{
+  "id": "it-discourse",
+  "type": "discourse",
+  "locationSet": {"include": ["ITA"]},
+  "languageCodes": ["it", "en"],
+  "order": 2,
+  "strings": {
+    "name": "Italian Discourse",
+    "description": "Il forum dove poter discutere di OpenStreetMap",
+    "url": "https://community.openstreetmap.org/t/italy-subcategory/2679/12"
+  },
```

```suggestion
  },
  "account": "it",
```

This would link to the category itself (https://community.openstreetmap.org/c/communities/it) rather than the creation discussion.